### ISSUE TITLE
Visual Studio fails to bind when attributes use const string interpolation

### ISSUE THREAD

**Issue by kapros** [SpecFlowOSS/SpecFlow.VisualStudio#194]
### SpecFlow Version

3

### Which test runner are you using?

NUnit

### Test Runner Version Number

N/A

### .NET Implementation

.NET 5.0

### Project Format of the SpecFlow project

Sdk-style project format

### .feature.cs files are generated using

SpecFlow.Tools.MsBuild.Generation NuGet package

### Test Execution Method

Visual Studio Test Explorer

### Issue Description

If your project uses the C#10 feature "constant interpolated strings" in your steps binding argument, the SpecFlow extension will no longer recognize the step as bound.
The tests seem to run just fine even from the VS test explorer.

### Steps to Reproduce

1. Create a SpecFlow project.
2. If reproducing before C#10 has been shipped, add this to your .csproj <PropertyGroup> node
    <LangVersion>preview</LangVersion>
3. In the StepDefinition attributes, add interpolation, for example create a const string with the whole text and insert that into the interpolated string.
4. Open the feature file with the affected. 

### Link to Repro Project

https://github.com/kapros/SpecFlowInterpolatedStrings

**Comment by gasparnagy** [comment]
@kapros @SabotageAndi Yes, this will work in the VS2022 extension, but not in the VS2019 extension. (I guess it would also work with Deveroom in VS2019 as it uses the same binding discovery mechanism as the VS2022 extension.)

As @kapros provided an repro, we should verify this with the VS2022 extension to make sure it works. I would keep the issue open until we did that.
